site/includes/brand.hbs

```diff
@@ -1,6 +1,6 @@
 {{#unless experimental}}
 	<div class="brand col-xs-8 col-sm-9 col-md-6">
-		<a href="https://www.canada.ca/{{language}}.html"><object type="image/svg+xml" tabindex="-1" data="{{assets}}/../{{site.theme}}/assets/sig-blk-{{language}}.svg"></object><span class="wb-inv"> {{{i18n "tmpl-gc-sig"}}}</span></a>
+		<a href="https://www.canada.ca/{{language}}.html"><object type="image/svg+xml" tabindex="-1" data="{{assets}}/../{{site.theme}}/assets/sig-blk-{{language}}.svg"></object><span class="wb-inv"> {{{i18n "tmpl-gc-sig"}}} / <span lang="{{#is page.language "en"}}fr">{{{i18n "tmpl-gc-sig" language="fr"}}}{{else}}en">{{{i18n "tmpl-gc-sig" language="en"}}}{{/is}}</span></span></a>
```

Comment:
@LaurentGoderre I don't see how I can simplify that logic any further without producing undesirable results. Only one is/else block is currently in place. Brand.hbs is included in both French/English pages and the FIP differs based on the current page's language.

English pages use a FIP containing "Government of Canada / Gouvernement du Canada", whereas French ones use "Gouvernement du Canada / Government of Canada". So I need to account for the switched order in the text alternative.

If you still believe there's a simpler way of coding it, what would you recommend?